package/cfg/config.yaml

```diff
@@ -25,6 +25,7 @@ master:
       - "hyperkube apiserver"
       - "hyperkube kube-apiserver"
       - "apiserver"
+      - "containerd"
```

Comment:
Is it possible that there might containerd process found on a node which is not a k3s node? is there any other k3s specific process or path that can be used?

Reply:
we tried binaries other than containerd and it doesnt work in k3s environment